docker/install-shared-deps.sh

```diff
@@ -62,3 +64,7 @@ curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh |
 curl https://cmake.org/files/v3.16/cmake-3.16.0-Linux-$(uname --machine).sh > /tmp/cmake.sh
 sh /tmp/cmake.sh --prefix=/usr/local --skip-license && /bin/rm /tmp/cmake.sh
 cmake --version
+
+# Install sccache
+cargo install --root /usr/local sccache
```

Comment:
Installing cargo and then compiling `sccache` adds both compilation time and bloat to the image.

Maybe it's better to download the binary directly from https://github.com/mozilla/sccache/releases with an md5sum check?